GoogleUtilities/AppDelegateSwizzler/GULAppDelegateSwizzler.m

```diff
@@ -940,8 +943,7 @@ - (void)application:(GULApplication *)application
   }
 }
 
-#if __IPHONE_OS_VERSION_MAX_ALLOWED >= 70000
-// This is needed to for the library to be warning free on iOS versions < 7.
+#if __IPHONE_OS_VERSION_MAX_ALLOWED >= 70000 && !TARGET_OS_WATCH
```

Comment:
No longer needed now?

Reply:
This is needed because of UIBackgroundFetchResult which is API_AVAILABLE(ios(7.0)). Somehow pod lib lint will fail if we ignore this check. I think something to do with the syntax API_AVAILABLE that checks below iOS 7 and it thinks UIBackgroundFetchResult is not recognized.

Reply:
Considering this is already there before my change, I can also add a todo for the owner to double check on this.